recoverMissingLumis.py

```diff
@@ -266,13 +266,14 @@ def getFiles(datasetName, runBlacklist, runWhitelist, blockBlacklist,
             for replica in replicaInfo["phedex"]["block"][0]["replica"]:
                 node = replica["node"]
                 cmsSites = siteDB.PNNtoPSN(node)
+                blockLocations.add(node.replace("_MSS","_Disk"))
```

Comment:
@amaltaro anything that is on tape would appear on disk with this, isn't that a bit risky ? shouldn't we just drop the block location if it is on tape ?